ext/http/01_http.js

```diff
@@ -282,6 +282,7 @@
                 resourceRid,
               );
               readableStreamClose(respBody); // Release JS lock.
+              core.tryClose(resourceRid);
```

Comment:
I'd do them the other way around, but otherwise LGTM

Reply:
Shouldn't both of them happen in `finally` block anyway?

Reply:
@bartlomieju  The `catch` block calls `reader.cancel()` which should close the resource.